src/backend/controllers/userController.js

```diff
@@ -182,8 +182,7 @@ exports.login = async (req: Request, res: Response) => {
 			return res
 				.status(401)
 				.json({ type: 'email-not-verified', msg: 'Your account has not been verified. ' + 
-					'Check your email for your verification token. ' +
-					'If your token has expired, request a new one.' });
+					'Check your email for verification, or request a new one.' });
```

Comment:
'Check your email **to verify your account**,...' would sound nicer but it's more a suggestion.

Reply:
I like this too.